migrations/20210615151625_fk.js

```diff
@@ -0,0 +1,30 @@
+exports.up = function (knex) {
+  const promises = [];
+  promises.push(knex.schema.table('appointments', (table) => {
+    table.foreign('psychologistId').references('psychologists.dossierNumber');
```

Comment:
Is this the same as `table.foreign('psychologistId').references('dossierNumber').inTable('psychologists');`?

Reply:
no idea, that's the code displayed in the doc
![image](https://user-images.githubusercontent.com/5389328/122366419-d3f5b180-cf5b-11eb-8aa1-5b4ee42ef76d.png)